js/sdk/src/components/NoSolWarning.js

```diff
@@ -0,0 +1,145 @@
+    setOpen(false)
+  }
+  return (
+    <Root>
+      <StyledModal
+        aria-labelledby="transition-modal-title"
+        aria-describedby="transition-modal-description"
+        open={open}
+        onClose={() => handleClose()}
+        closeAfterTransition
+        BackdropComponent={Backdrop}
+        BackdropProps={{
+          timeout: 500,
+        }}
+      >
+        <Fade in={open}>
+          <StyledPaper>
+            <>
+              <StyledTypography variant="h4" sx={{ paddingBottom: '16px' }}>
+                {` You do not have enough ${currency} in your wallet to ${actionText}.`}
+              </StyledTypography>
+              <StyledTypography component="p" gutterBottom>
+                {`${requiredSol} ${currency} is required to ${actionText}.`}
+              </StyledTypography>
+              <StyledTypography component="p" gutterBottom>
+                {`You currently have 0
+            ${currency} in your wallet.`}
+              </StyledTypography>
+              <StyledTypography component="p" gutterBottom>
+                {`Please add more ${currency} to your wallet to ${actionText}.`}
+              </StyledTypography>
+              <StyledTypography
+                component="p"
+                gutterBottom
+                sx={{ display: 'flex', flexDirection: 'row' }}
+              >
+                {`For any questions, please reach out to us at`}
+                &nbsp;
+                <Link href="mailto:contact@ninaprotocol.com">
+                  <a
+                    target="_blank"
+                    rel="noreferrer"
+                    style={{ margin: '0px', textDecoration: 'none' }}
+                  >
+                    <ContactTypography component="p">
+                      {`contact@ninaprotocol.com`}
+                    </ContactTypography>
+                  </a>
+                </Link>
+                {`.`}
+              </StyledTypography>
+              <Button
+                style={{ marginTop: '15px' }}
+                color="primary"
+                variant="outlined"
+                onClick={handleClose}
+              >
+                <Typography>Got it</Typography>
+              </Button>
+            </>
+          </StyledPaper>
+        </Fade>
+      </StyledModal>
+    </Root>
+  )
+}
+
+const Root = styled('div')(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  width: '100%',
+}))
+
+const StyledModal = styled(Modal)(() => ({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+}))
+
+const StyledPaper = styled(Paper)(({ theme }) => ({
+  backgroundColor: theme.palette.background.paper,
+  border: '2px solid #000',
+  boxShadow: theme.shadows[5],
+  padding: theme.spacing(2, 4, 3),
+  width: '40vw',
+  maxHeight: '90vh',
+  overflowY: 'auto',
+  zIndex: '10',
+  display: 'flex',
+  flexDirection: 'column',
+}))
+
+const StyledTypography = styled(Typography)(() => ({
+  marginBottom: '20px',
```

Comment:
`Typography` accepts a prop called `gutterBottom` that we use to do this. 

`20px` is not a margin size that we use